satellite/marketing/marketingweb/server.go

```diff
@@ -5,17 +5,24 @@ package marketingweb
 
 import (
 	"context"
+	"html/template"
 	"net"
 	"net/http"
+	"runtime"
+	"strings"
 
 	"github.com/zeebo/errs"
 	"go.uber.org/zap"
 	"golang.org/x/sync/errgroup"
+	"github.com/gorilla/mux"
 )
 
 // Error is satellite marketing error type
-var Error = errs.Class("satellite marketing error")
-
+var (
+	Error = errs.Class("satellite marketing error")
+	_, fp, _, _ = runtime.Caller(0)
```

Comment:
That file path will be bound to your local code location, that may not work for precomputed binaries. Also you set hard rule for path of your static files to be in folder where server.go was during compilation. But I am not sure if we care about making static files configurable this much